scanapi/reporter.py

```diff
@@ -42,4 +44,5 @@ def _build_context(results):
             "project_name": settings.get("project_name", ""),
             "results": results,
             "session": session,
+            "scanapi_version": get_distribution("scanapi").version,
```

Comment:
@hebertjulio hey 👋🏾 Can you update the docstring to the method mentioned [here](https://github.com/scanapi/scanapi/wiki/First-Pull-Request#7-make-your-changes) in the wiki. Let's slowly self document our code. 🙌🏾 

Something like 
```
    @staticmethod
    def _build_context(results):
        """Formating context dict of values required to render template.
        Args:
            results [type]: Fill me!
        Returns:
            [dict]: values required to render template.
        """
        return {
            "now": datetime.datetime.now().replace(microsecond=0),
            "project_name": settings.get("project_name", ""),
            "results": results,
            "session": session,
            "scanapi_version": get_distribution("scanapi").version,
        }
```

I am not sure what would be the type of results and what would be the appropriate short description for it. 

Rest of the PR LGTM. Thank you for the help. 😄

Reply:
Hey @Pradhvan,

I think results is a list of request, that right @camilamaia ?

Reply:
@hebertjulio, results is a generator object! Please, take a look at this answer, where you can see a more detailed explanation:

https://github.com/scanapi/scanapi/pull/384#discussion_r644227563